styles/components/modal/_variables.scss

```diff
@@ -0,0 +1,36 @@
+@import '@addepar/style-toolbox/core-variables/colors';
+@import '@addepar/style-toolbox/core-variables/spacing';
+@import '@addepar/style-toolbox/core-variables/typography';
+@import '@addepar/style-toolbox/utilities/typography/mixins';
+@import '@addepar/style-toolbox/utilities/typography/fonts';
+@import '@addepar/style-toolbox/core-variables/borders';
+@import '@addepar/style-toolbox/elements/inputs/button/index';
+
+// Text
+$adde-modal-font-family:                $font-family-sans-serif;
+$adde-modal-header-text-color:          $text-white;
+$adde-modal-header-font-size:           18px;
+$adde-modal-header-font-weight:         $font-normal;
+
+// Backgrounds
+$adde-modal-background:                 $background-white;
+$adde-modal-overlay-background:         rgba($background-dark, 0.5);
+$adde-modal-header-background:          $background-dark;
+$adde-modal-hover-close-background:     #4d4d4d;
```

Comment:
is there a core color we can use instead?

Reply:
The next lightest color ($background-medium-dark) looks like this: 
![image](https://user-images.githubusercontent.com/13475895/40999387-8e3813fa-68bf-11e8-9819-e2fa0372ec29.png)

This color is what we use in the ember-widgets modal now, I could make a new color variable for it maybe? (`$background-dark-hover` ?)

Reply:
Missed this response!
Can just do something like this 
`$adde-modal-hover-close-background:  lighten($adde-modal-header-background, 5%);`
or whatever looks good.